README.md

```diff
@@ -386,6 +387,30 @@ Creates a connection invitation for a user to create a relationship with another
 | result  | 'success' | String  |
 | message  | 'Connection invitation was successfully sent' | String  |
 
+### Connection v2 Invite
+
+```typescript
+pillarSdk.connectionV2.invite(inputParams)
+```
+
+Creates a connection invitation for a user to create a relationship with another contact.
+
+**Input Parameters**
+
+| Name  | Description | Type | Required |
+| ------------- | ------------- | ------------- | ------------- |
+| targetUserId  | The Contact User Identifier. | UUID  | Required  |
+| sourceIdentityKey  | Generated source identity key. | String  | Required  |
```

Comment:
Let's add a better description here for the identity keys:
Generated source identity key for the source user.
Generated target identity key for the source user.